src/modules/auth/Auth.page.tsx

```diff
@@ -0,0 +1,8 @@
+import useLazyLoadResourceHook from '@shared/hooks/lazyLoadResource.hook';
+import { SignUp } from './components/SignUp';
+
+export default function Auth(): JSX.Element {
```

Comment:
Uopste nisam fan `export default` kombinacije, jer to omogucava da istu stvar importujes pod apsolutno razlicitim imenima... Ali u ovom slucaju, jer se Auth ucitava kao stranica kroz `lazy loading`, zgodnije je da se tako exportuje komponenta. Tako da ovde je ok, ali za druge stvari ce da koristimo "cist" `export`